### PR TITLE
CosanlabToolbox Cleanup

### PR DESCRIPTION
**Pull request by danweflen**
These commits change a bunch of permissions, remove files that should have been excluded by the .gitignore, and do a bunch of other miscellaneous cleaning.

### CODE REVIEW

**Comment by ljchang** [comment]
Thanks @danweflen for all the work cleaning up.  Where were those changes to the couples paradigm from?  i thought I had the most up to date version on github.

File: Matlab/DecisionFunctions/comp_model.m [hunk 1]
```diff
@@ -471,28 +472,28 @@
...
+            % minimized value, and number of subjects.
             % -------------------------------------------------------------------------
             
             sprintf(['Summary of Model: ' obj.model ...
                 '\n-----------------------------------------' ...
                 '\nAverage Parameters:\t' num2str(nanmean(obj.params(:,2:length(obj.param_min) + 1))) ...
                 '\nAverage AIC:\t\t' num2str(nanmean(obj.params(:,end - 1))) ...
                 '\nAverage BIC:\t\t' num2str(nanmean(obj.params(:,end))) ...
-                '\nAverage ' obj.esttype ':\t\t' num2str(nanmean(obj.params(:,end-3))) ...
+                '\nAverage ' obj.esttype ':\t\t' num2str(nanmean(obj.params(:,end-2))) ...
```
Comment (ljchang) [inline Matlab/DecisionFunctions/comp_model.m]: @danweflen: Are you sure this is right?  I vaguely remember fixing a random bug here awhile back.

Comment (danweflen) [inline Matlab/DecisionFunctions/comp_model.m]: Feel free to get rid of it.

Comment (ljchang) [inline Matlab/DecisionFunctions/comp_model.m]: Was just curious if you found a bug or were pushing old code that had been changed.

**Comment by danweflen** [comment]
You should probably just cherry-pick commits  c18b178, 32d296d, and  3e61e5e, the other ones I'm not sure about.

File: Matlab/DecisionFunctions/comp_model.m (hunk 1 above)
Comment (ljchang) [inline Matlab/DecisionFunctions/comp_model.m]: weird it was changed #c265b55c75184684d70fdcade73215f40864cdfd

**Comment by ljchang** [comment]
I bet it was when you merged your version with the new version.  It's probably not going to modify those files as they are already in the current version.  I can't figure out how to only selectively approve specific commits.

**Comment by danweflen** [comment]
I can do it. Close this pull request, and I'll make another one with only the good commits in it.

**Comment by ljchang** [comment]
thanks @danweflen!

**Pull request closed without merging**